schemas/acquisitions/vendorEdiOrdersExportConfig.json

```diff
@@ -60,6 +60,14 @@
           "CSV",
           "EDI"
         ]
+      },
+      "claimPieceIds": {
+        "description": "Claim piece ids. Applicable only for Claiming integration type",
+        "type": "array",
+        "items": {
+          "description": "UUID of the piece",
+          "type": "string"
+        }
```

Comment:
`"format": "uuid"` can be added here

Reply:
if we add it we will have List<UUID> instead of List<String>